annotation-processing/src/test/java/com/exactpro/epfast/annotation/processing/FastProcessorTest.java

```diff
@@ -5,11 +5,24 @@
 import org.junit.jupiter.api.Test;
 
 import javax.tools.StandardLocation;
+import java.nio.charset.Charset;
 
 class FastProcessorTest {
+    @Test
+    void testServiceCreation() {
```

Comment:
The check that service file is created should be added to all other tests instead.

Reply:
@lashamez you forgot to do it